docs/Datadog.Trace.Annotations/README.md

```diff
@@ -0,0 +1,13 @@
+# Datadog.Trace.Annotations NuGet package
```

Comment:
Should we also add a note to the Datadog.Trace readme, mentioning the existence of this package?

Reply:
Good idea, I'll draft up some changes that you can review

Reply:
Updated in 5154c99345a8b0a17d562567dc9508d91affb66f. WDYT?